src/Microsoft.Health.SqlServer/IAccessTokenHandler.cs

```diff
@@ -1,20 +1,22 @@
-﻿// -------------------------------------------------------------------------------------------------
+// -------------------------------------------------------------------------------------------------
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using Microsoft.Health.SqlServer.Configs;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.Health.SqlServer;
 
 public interface IAccessTokenHandler
 {
+    SqlServerAuthenticationType AuthenticationType { get; }
```

Comment:
Similar to AuthenticationType. Should we create a method like AzureResourceProperty which must be implemented by the base class?